apps/web/src/components/pages/editor/education-plugin-examples.tsx

```diff
@@ -43,7 +43,7 @@ function createBoxExample(title: string, content: string, type: BoxType) {
   } as EditorBoxDocument
 }
 
-export const boxExample = (
+export const BoxExample = (
```

Comment:
I don't really care here but for context: I lowercased them because they are not react components.

Reply:
> Feels like a constant to me, that we are PascalCasing

who, where? E.g. this folder is full of constant definitions in lower case.